rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -150,7 +150,7 @@ def __init__(
         decoder that should be included for this decoder modules.
         """
         self.database = database
-        self.misc_counterparties = [CounterpartyDetails(identifier=CPT_GAS, label='gas', image='gas.svg')] + misc_counterparties  # noqa: E501
+        self.misc_counterparties = [CounterpartyDetails(identifier=CPT_GAS, label='gas', icon='fire-line')] + misc_counterparties  # noqa: E501
```

Comment:
is there any difference to before in production?

Reply:
yes the icons wasn't showing in production because the image doesn't exist